libs/libcommon/src/libcommon/queue.py

```diff
@@ -116,6 +119,21 @@ class JobQueryFilters(TypedDict, total=False):
     difficulty__lte: int
 
 
+PA_SCHEMA = Schema(
+    {
+        "_id": bson.ObjectId,
+        "type": pa.string(),
+        "dataset": pa.string(),
+        "revision": pa.string(),
+        "config": pa.string(),
+        "split": pa.string(),
+        "priority": pa.string(),
+        "status": pa.string(),
+        "created_at": pa.timestamp("ms"),
```

Comment:
I tried to align with the old mapping "ns," but Pymongoarrow throws an error:
`TypeError: PyMongoArrow only supports millisecond temporal resolution compatible with MongoDB's UTC datetime type.`
I verified, and also, we support up to milliseconds in the Mongo Collection, so I think it is not a big deal.